CyrillicFilters/RussianFilter/sections/specific.txt

```diff
@@ -227,6 +227,8 @@ comandir.com#?##dle-content > noindex:has(> div.hblock > div[id*="ScriptRoot"])
 muz-tv.ru##.x-popup-ad-short
 marieclaire.ru##.article__block_type-productSet
 pc.ru##div[class^="advert_"]
+patee.ru##div:has(> .yandex-rtb)
```

Comment:
I would prefer more specific rule, because it might causes incorrect blocking and be inefficient.
It seems that on mobile page it hides all `#teasers1` elements.
<details><summary>Screenshot</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/29142494/bdb1c860-8fd2-46b9-aea3-78492cb84b3d)

</details>

Maybe something like this:
```adblock
patee.ru###top > div[style]:has(> .yandex-rtb)
patee.ru##.colC > .placeR:has(> .yandex-rtb)
```